examples/hselect.rs

```diff
@@ -0,0 +1,28 @@
+extern crate session_types;
+
+use session_types::*;
+
+fn main() {
+    let (tcs, rcs) = session_channel();
+    let (tcu, rcu) = session_channel();
+
+    let receivers = vec!(rcs, rcu);
+
+    let _ = tcs.send("Hello, World from TCS!".to_string()).close();
+
+    let (ready, mut rest) = hselect(receivers);
+
+    let (to_close, s) = ready.recv();
+    println!("Got a response: \"{}\"", s);
+    to_close.close();
+
+    let _ = tcu.send("Hello, World from TCU!".to_string()).close();
```

Comment:
I think all the call to `close()` do not need the `let _ = …`, ie

```
tcu.send("Hello, World from TCU!".to_string()).close();
```
should suffice.

(This applies in a few places.)